### ISSUE TITLE
bug

### ISSUE THREAD

**Issue by drbenvincent** [brian-lau/MatlabProcessManager#3]
I'm using this in conjunction with MatlabStan. When I call the compile method on the StanModel object, I get this error

```
Error using processManager/start (line 411)
Looks like command doesn't exist. Check spelling or path?

Error in processManager/set.command (line 215)
            self.start();

Error in processManager (line 178)
         self.command = p.Results.command;

Error in StanModel/compile (line 846)
         p = processManager('id','compile',...
```

I've been using MatlabProcessManager and MatlabStan happily for ages. I suspect the bug was introduced in the last commit to this repo. Would appreciate any pointers.

**Comment by brian-lau** [comment]
Sorry bout that. I think I know what the problem is. Just to be sure, can you tell me what OS you are running on?

**Comment by brian-lau** [comment]
I pushed a fix up in v0.5.1. Let me know if this doesn't work.

**Comment by drbenvincent** [comment]
Yep, working again for me. Thanks for the speedy response.
